pdsClient/api.go

```diff
@@ -217,3 +217,12 @@ type ListRecordsResult struct {
 	Results   []ListedRecord `json:"results"`
 	LastIndex int            `json:"last_index"`
 }
+
+type BatchGetRecordsRequest struct {
+	RecordIDs   []string `json:"record_ids"`
+	IncludeData bool     `json:"include_data"`
+}
+
+type BatchGetRecordsResult struct {
```

Comment:
What happens a single record fails?

Reply:
A record "failure" means either the record doesn't exist or the reader can't read it so it won't come back in the results, its the responsibility of the caller to determine whether that was the case, and which one didn't get returned(by doing some set operations on the requested id's versus the returned id's), and then call non batch get for the record(s) of interest to determine whether 404 or 401'd